flexmeasures/data/models/planning/utils.py

```diff
@@ -283,3 +291,74 @@ def idle_after_reaching_target(
     else:
         schedule[schedule.cumsum() < first_target] = 0
     return schedule
+
+
+def get_series_from_sensor_or_quantity(
```

Comment:
I think that the function should fill the missing values in case that the sensor data is incomplete. However, with the addition of a custom `default_value` (default = np.nan) and making the `default_value_attribute`  optional, we cover more cases.

For example: If `default_value_attribute` is None and `default_value` is np.nan, we would fill the missing values of `quantity_or_sensor` with nan.

```python
get_series_from_sensor_or_quantity(
    quantity_or_sensor = capacity_sensor,
    actuator = power_sensor,
    target_unit = "MW",
    query_window = (start, end),
    resolution = power_sensor.event_resolution,
) 
```

Moreover, If no default value is passed, no np.nan value will be filled and the fallback logic can leave outside.